Weather app

thsvr opened the pull request.

Oghenebrume50 commented on .eslintrc.json:
```diff
@@ -10,14 +10,9 @@
-        "@typescript-eslint"
-    ],
     "rules": {
     }
```
No change should be made to eslintrc file ![see this](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/linter_config.png)
[see correct file](https://github.com/microverseinc/linters-config/blob/master/javascript/.eslintrc.json)

thsvr commented:
Hi TSE :smile: Hope you doing well :smile: 

:v: As requested by Raphael, I fixed the eslint file. 

:v: About the request: _"There should be a change in the appearance for every location searched"_ You can see the icon changing every time we change the location.

![weather-app](https://user-images.githubusercontent.com/49765920/76774335-8865be00-6782-11ea-8e34-3a85fdb279ba.gif)


:v: About the request to take different data for Celsius and Fahrenheit, I just used the unique MAIN DATA which is available on the API to access the temperature:

![api-data](https://user-images.githubusercontent.com/49765920/76774555-d24ea400-6782-11ea-975e-f5477b4d1633.png)

:v: About the question from Raphael:  _According to the screenshot above, there are two buttons toggle why is this so?_
As I mentioned in the previous PR, I added an extra tool to switch temperature between Celsius and Kelvin. 

Can you please check if my project is able to be approved and provide feedback?

Thank you in advance,

Thaís Vieira

MauricioRobayo commented on src/index.js:
```diff
@@ -0,0 +1,109 @@
+/* eslint-disable prefer-destructuring */
+/* eslint-disable no-undef */
+/* eslint-disable no-return-assign */
+/* eslint-disable arrow-parens */
```
🤨 You are disabling Eslint rules all over the place 😅... Please don't do that, you should try to stick to the rules provided!

MauricioRobayo commented on package.json:
```diff
@@ -23,8 +24,29 @@
+    "eslint": "^6.8.0",
+    "eslint-config-airbnb-base": "^14.1.0",
+    "eslint-plugin-import": "^2.20.1",
     "webpack": "^4.41.6",
```
The main purpose of Webpack is so that you can split your code into different modules, but you have everything in a single file. You should try to modularize your code.

MauricioRobayo commented on src/index.js:
```diff
@@ -0,0 +1,109 @@
+  return { getWeather, getTemp, getPlace };
+};
+const dataArray = [];
+searchForm.addEventListener('submit', event => {
```
This is a huge block of code inside the event listener, you could try to refactor/extract some pieces of it into reusable functions, that's going to make it more readable/maintainable/debuggable.
